### ISSUE TITLE
Cannot login, blank red box, page shows JS errors

### ISSUE THREAD

**Issue by adityavb** [awslabs/web-client-for-aws-transfer-family#11]
When attempting to login with correct or incorrect credentials a blank red box appears above the login button. Browser console shows error as screenshot attached.
![Screen Shot 2022-03-07 at 11 45 00 AM](https://user-images.githubusercontent.com/65777183/157345553-ab2316be-7e0b-4991-9b50-af68fac727ec.png)

**Comment by adityavb** [comment]
ECS logs:


[2022-03-08 16:31:28 +0000] [6] [INFO] Starting gunicorn 20.0.4
--
[2022-03-08 16:31:28 +0000] [6] [INFO] Listening at: http://0.0.0.0:80 (6)
[2022-03-08 16:31:28 +0000] [6] [INFO] Using worker: sync
[2022-03-08 16:31:28 +0000] [8] [INFO] Booting worker with pid: 8
[2022-03-08 16:31:28 +0000] [9] [INFO] Booting worker with pid: 9
[2022-03-08 16:31:28 +0000] [10] [INFO] Booting worker with pid: 10
sftpwebclientlogger  INFO  jwt_secret_key found in the environment variable:3dxxxxbe631
sftpwebclientlogger  INFO  jwt_secret_key found in the environment variable:3dxxxxbe631
sftpwebclientlogger  INFO  jwt_secret_key found in the environment variable:3dxxxxbe631

**Comment by rathialok901** [comment]
I am having the exact same problem, the Cognito part has been correctly setup as I can do a manual test authentication with Lambda and it returns successful auth and the directory Mapping from DynamoDB.

**Comment by rathialok901** [comment]
@adityavb Try Connecting to the SFTP Endpoint and see if that is working as in the backend the web is opening an SFTP connection and if that doesnt work then Web Logons will fail too.

**Comment by adityavb** [comment]
@rathialok901 yes I am able to login to the server directly using OpenSSH, FileZilla and WinSCP

**Comment by rathialok901** [comment]
@adityavb Yes I have web logins working but the UI experience is not great and is pretty slow. Can you switch your browser to developer mode and see what is happening to your request and that could help point out the problem.

**Comment by sapra5379** [comment]
Team - I am deploying this solution in Bahrain region. I wanted to understand what is the use of AWS Lambda@Edge and why it is need to be deployed only in Virginia Region.
Kindly let me know at the earlies @rathialok901 or @adityavb

**Comment by sapra5379** [comment]
@rathialok901  @adityavb  @ajorg-aws - I am able to set it up, However - I am getting red screen while login. Can anyone of you available for a quick screenshare.
Need to check dynamoDB and deployment.

**Comment by rathialok901** [comment]
> Team - I am deploying this solution in Bahrain region. I wanted to understand what is the use of AWS Lambda@Edge and why it is need to be deployed only in Virginia Region. Kindly let me know at the earlies @rathialok901 or @adityavb

That is because CloudFront, although a Global Service has its control plane in us-east-1 and that is why you are deploying the Lambda@edge in that region.

**Comment by rathialok901** [comment]
> @rathialok901 @adityavb @ajorg-aws - I am able to set it up, However - I am getting red screen while login. Can anyone of you available for a quick screenshare. Need to check dynamoDB and deployment.

I can work with you tommorrow to look at what is going on, In what timezone are you available.

**Comment by sapra5379** [comment]
I am getting authentication error, it must be something wrong at my DynamoDB mapping.

Failed to authenticate user asapra with Cognito

**Comment by dagarkatyal** [comment]
I deployed everything twice but I am getting CORS errors when I click login button.
Any thoughts? What am i missing here? any account level setting?
here is the screenshot..

<img width="1322" alt="Screen Shot 2022-07-22 at 12 57 11 AM" src="https://user-images.githubusercontent.com/10888487/180373714-10c95221-f4a8-4bf0-a1ec-741b12939911.png">

**Comment by nagarjuna-jammula** [comment]
> @rathialok901 @adityavb @ajorg-aws - I am able to set it up, However - I am getting red screen while login. Can anyone of you available for a quick screenshare. Need to check dynamoDB and deployment.

Hello, we have deployed the solution and getting same issue. Is there already a solution for this ?

**Comment by nagarjuna-jammula** [comment]
Hello,

We have deployed the solution. User created, dynamo DB mapping is completed but facing the same issue as original. It shows red box below when trying to login.

Is there already a solution for this ?

**Comment by lalithprasad12** [comment]
Hi, Any update on the issue. We are facing the same problem.
